### ISSUE TITLE
[Onboarding] Update Tab selector colors in Seed Input page view

### ISSUE THREAD

**Issue by alexandraB99** [status-im/status-desktop#5612]
# Bug Report

## Description
Tabs selectors colors in Seed Input page view are not the same as the ones in the design

1. Choose to create new keys or import existing account
2. Choose to import seed phrase

#### Expected behavior
Tabs selectors colors in Seed Input page view should be according to design
![image](https://user-images.githubusercontent.com/31625338/166475130-b836d7a2-ce37-45a9-bbd6-45f934144ec8.png)

[figma reference](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=2292%3A291035)

#### Actual behavior
Tabs selectors colors in Seed Input page view are not the same as the ones in the design

### Additional Information

- Status desktop version:
master
- Operating System:
macOS

**Comment by MishkaRogachev** [comment]
I've checked other pages in figma and find that we have several places with such colors:
![image](https://user-images.githubusercontent.com/2522130/166706441-88e4dce5-4a20-4b56-8cab-cc2930114046.png)
* Should i change only this page or common component is StatusQ?

* And what solution i should choose:
1. Add ability to custamize colors in StatusSwitchTabBar/StatusSwitchTabButton
2. Add enum-based style property to StatusSwitchTabBar

**Comment by PascalPrecht** [comment]
I think the reason what's on your screenshot looks different than on @alexandraB99 is that these are coming from two different themes - light an dark.

I don't think an enum based style property is necessary at this point, given that there's only one variant of this component used throughout the design. It does have to function across dark and light theme though.

**Comment by MishkaRogachev** [comment]
Some experiments here: https://github.com/status-im/StatusQ/pull/650
I'm a bit confused with theme colors. Why we don't use colors exacly how they are in Figma?

**Comment by PascalPrecht** [comment]
> I'm a bit confused with theme colors. Why we don't use colors exacly how they are in Figma?

I guess this depends on what exactly you're referring to.
Current StatusQ uses the color palette definition of the previous design system.
Those colors were exactly the ones used in figma (and mostly are still the same).

**Comment by PascalPrecht** [comment]
> To be sure I need a way to see Figma in dark theme.

Unfortunately, not all screen designs are done in dark theme. But they use the colors defined in the color palette here, in which you can map each color to its dark theme equivalent: https://www.figma.com/file/eghXX5JkzSM14Wtgyx1S2u/%F0%9F%8E%A8-Design-System-%E2%8E%9C-Desktop-Backup?node-id=1%3A10

Because colors don't always match exactly in both themes (for example what's 10% blue in light mode might be 20% blue in dark mode), we have a naming system that doesn't use concrete color names.

See below:

![Screenshot from 2022-05-06 07-43-33](https://user-images.githubusercontent.com/445106/167073992-2dff1c1d-e51c-422a-8027-2aa4db1e48f5.png)

Notice the names (Primary, Base, Danger, Success, ... etc) and the number matrix. If you're looking for 10% light blue in light mode, you can see that it's the color located at "Primary:3" in light mode, which maps to `Theme.palette.primaryColor3` in QML.

Also, notice how "Primary:3" is **not** 10% light blue, but 20% light blue. 
Our color palette system in QML takes this already into account.

Bottom line: You don't need to introduce new colors, they already exist here: https://github.com/status-im/StatusQ/blob/master/src/StatusQ/Core/Theme/ThemePalette.qml#L40-L91

And they are specifically defined for individual themes here: 

- https://github.com/status-im/StatusQ/blob/master/src/StatusQ/Core/Theme/StatusLightTheme.qml#L101-L146
- https://github.com/status-im/StatusQ/blob/master/src/StatusQ/Core/Theme/StatusDarkTheme.qml#L97-L148

For more info on the reasoning of everything, checkout: https://github.com/status-im/StatusQ/issues/3

**Comment by MishkaRogachev** [comment]
Thanks so much for detailed answer, it help me a lot! 

But colors used in Figma does not match existing in palette. For example i need Light Desktop/Blue, and it is similar in dark theme. The closest color in out palette is Primary1 in light theme but it is differnt to figma. Here colors used in figma:
| Item | Dark | Light |
| ----------- | ----------- | ----------- |
| Active tab button | #4360df | #4360df |
| Active tab text | #ffffff | #ffffff |
| Unactive tab backgound | #262d3f | #edf0fc |
| Unactive tab text | #4360df | #6d84e6 |

![image](https://user-images.githubusercontent.com/2522130/167085006-d068b1a4-c544-4caa-944f-383f1d364824.png)
And whom should i ping for access to Figma? =)

**Comment by PascalPrecht** [comment]
For now, I think it's best to stick to the color system we already have to preserve consistency /cc'ing @John-44 to confirm
Otherwise, you can always hardcode the color codes for each theme (kinda what you did already in the parent theme file.

As for access to figma, I do not know. Would assume @John-44 or @holix
